x/incentives/keeper/gauge.go

```diff
@@ -196,40 +204,42 @@ func (k Keeper) GetGaugeFromIDs(ctx sdk.Context, gaugeIDs []uint64) ([]types.Gau
 	return gauges, nil
 }
 
-// GetGauges returns gauges both upcoming and active.
+// Returns both upcoming and active gauges.
+// TODO: This also returns finished gauges right? Otherwise would be same as GetNotFinishedGauges
```

Comment:
TODO: This also returns finished gauges right? Otherwise would be same as GetNotFinishedGauges

Reply:
Nice catch - yes, it seems as though this gets all gauges (active, upcoming, and finished) since it uses this key for its iterator:
https://github.com/osmosis-labs/osmosis/blob/8f018a6c638b64912c1b22c7df04f84abe667ba0/x/incentives/types/keys.go#L28-L29